doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -467,34 +467,34 @@ script as root::
 
   $ sudo src/spdk/scripts/setup.sh
 
-Then you need to specify NVMe device's device selector here with "spdk:" prefix for
-``bluestore_block_path``.
+You will need to specify the subject NVMe device's device selector with
+the "spdk:" prefix for ``bluestore_block_path``.
 
-For example, users can find the device selector of an Intel PCIe SSD with::
+For example, you can find the device selector of an Intel PCIe SSD with::
 
   $ lspci -mm -n -D -d 8086:0953
 
 The device selector always has the form of ``DDDD:BB:DD.FF`` or ``DDDD.BB.DD.FF``.
 
 and then set::
 
-  bluestore block path = spdk:0000:01:00.0
+  bluestore_block_path = spdk:0000:01:00.0
 
 Where ``0000:01:00.0`` is the device selector found in the output of ``lspci``
 command above.
 
-If you want to run multiple SPDK instances per node, you must specify the
-amount of dpdk memory size in MB each instance will use, to make sure each
+To run multiple SPDK instances per node, you must specify the
+amount of dpdk memory in MB that each instance will use, to make sure each
 instance uses its own dpdk memory
 
-In most cases, we only need one device to serve as data, db, db wal purposes.
-We need to make sure configurations below to make sure all IOs issued under
-SPDK.::
+In most cases, a single device can be used for data, DB, and WAL, ie. they are
+*colocated*. Be sure to enter the below settings to ensure that all IOs are
+issued through SPDK.::
 
   bluestore_block_db_path = ""
   bluestore_block_db_size = 0
   bluestore_block_wal_path = ""
   bluestore_block_wal_size = 0
 
```

Comment:
@optimistyzy @neha-ojha @zdover23 Please review the below changes to the last sentence in this file.  The existing text isn't very clear.  After research I'm _fairly_ confident that my wording is both accurate and precise, but I'd like some confirmation as I am not conversant with SPDK.

Reply:
@anthonyeleven This might just be an instance of that legendary and irresistible passion to edit the draft of someone else that H.G. Wells spoke of, but I'd alter this slightly. 
```suggestion
In most cases, a single device can be used for data, DB, and WAL. This means that these functions can be *colocated*. 
```

Reply:
Ack.  This is a case where I add in a term for the express purpose of introducing the reader to vernacular that might be found in the wild, e.g. the mailing lists.  I've tweaked this, PTAL